kerby-kerb/kerb-admin-server/src/main/java/org/apache/kerby/kerberos/kerb/admin/server/kadmin/AdminServerHandler.java

```diff
@@ -199,6 +199,42 @@ private ByteBuffer handleGetprincsReq(LocalKadmin localKadmin, XdrFieldInfo[] fi
             return responseError;
         }
     }
+    
+    private ByteBuffer handleExportKeytabReq(LocalKadmin localKadmin, XdrFieldInfo[] fieldInfos) throws IOException {
+        String principals = ((String) fieldInfos[2].getValue());
+        
+        if (principals != null) {
+            List<String> princList = stringToList(principals);
+            if (princList.size() != 0) {
+                LOG.info("Exporting keytab file for " + principals + "...");
+                File path = new File("/tmp/" + System.currentTimeMillis());
```

Comment:
I just happened to notice this. It is not the correct temporary directory on Windows and it is also not secure. Consider using `File.createTempFile()` instead.

Reply:
I have fixed it by using Files.createTempDirectory("ktadd").